spec/std/file_spec.cr

```diff
@@ -126,7 +130,9 @@ describe "File" do
       ex = expect_raises(Errno, /Error determining size/) do
         File.empty?("#{__FILE__}/")
       end
-      ex.errno.should eq(Errno::ENOTDIR)
+      {% unless flag?(:win32) %}
+        ex.errno.should eq(Errno::ENOTDIR)
```

Comment:
These are because `expect_raises` is stubbed out on `win32`, so the return type changes.